src/Nethermind/Nethermind.Merge.Plugin/BlockProduction/Boost/BoostBlockImprovementContext.cs

```diff
@@ -34,7 +35,7 @@ public class BoostBlockImprovementContext : IBlockImprovementContext
         _boostRelay = boostRelay;
         _stateReader = stateReader;
         _cancellationTokenSource = new CancellationTokenSource(timeout);
-        CurrentBestBlock = currentBestBlock;
+        Block = currentBestBlock;
```

Comment:
CurrentBestBlock was better name. Why did we change it?

Reply:
idk @LukaszRozmej

Reply:
if no good reasons please revert it